group/basic/main.tf

```diff
@@ -1,7 +1,8 @@
 # AWS Autoscaling group
 
 ## Creates security group
-resource "aws_security_group" "sg_asg" {
+resource "aws_security_group" "asg-sg" {
+  name_prefix = "${var.stack_item_label}-"
```

Comment:
maybe `${var.stack_item_label}-asg-sg` to match the `stack_item_label` usage we have elsewhere, making it less likely we'll forget to append the naming convention?